### PR TITLE
Updated README.md file to run the app with custom property file & env variables

### PR DESCRIPTION
**Pull request by shubhamparikh927**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +274,13 @@ Server will then be accessible at http://localhost:8080/ and eg. http://localhos
           refuse_to_fetch_third_party_urls: false
           fhir_version: R4
 ```
+Also with custom property file and enviornment variables , use below docker command, replace <values> with actual values
```
Comment (hankwallace) [inline README.md]: Suggest the following wording:
"You can use a custom property file that utilizes environment variables for many configuration properties. For example, to use the `application-custom.yaml` file. Replace <values> with actual values."

File: README.md [hunk 2]
```diff
@@ -274,6 +274,13 @@ Server will then be accessible at http://localhost:8080/ and eg. http://localhos
...
+
+```bash
+./build-docker-image.sh && docker run -p 8080:8080 -e APIKEY=<apikey> -e APIKEY_ENABLED=<true/false> -e datasource.driver=<datasource_driver> -e datasource.password=<datasource_password> -e datasource.url=<jdbc_url> -e datasource.username=<datasource_password> -e enable_web=<true/false> -e fhir_version=<DSTU2/DSTU3/R4> -e hapi.fhir.server_address=<server_address>/fhir/ -e hibernate.dialect=<hibernate_dialect> -e OAUTH_ENABLED=<true/false> -e OAUTH_URL=<oauth_server_url> -e reuse_cached_search_results_millis=<milliseconds_value_to_reuse_cached_search_results> -e spring.config.location='</usr/local/tomcat/webapps/ROOT/WEB-INF/classes/application-custom.yaml>' -e subscription.resthook.enabled=<true/false> -e subscription.websocket.enabled=<true/false> -e url_pattern=</fhir/*> hapi-fhir/hapi-fhir-jpaserver-starter:latest
```
Comment (hankwallace) [inline README.md]: This is a very long line - please use the `\` line continuation character to make this more readable.

**Review by hankwallace — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shubhamparikh927) [reply README.md]: Ok

**Review by hankwallace — APPROVED** [review]
Looks good

**Pull request merged**